Subject: [PATCH] password change patch test
---
Index: app/web/sites/default/settings.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/web/sites/default/settings.php b/app/web/sites/default/settings.php
--- a/app/web/sites/default/settings.php	(revision 2d9e2af8979e223b1940ac9e5abec6c3fa6ba00b)
+++ b/app/web/sites/default/settings.php	(date 1692359941228)
@@ -95,7 +95,7 @@
         array (
           'database' => 'composer_tests',
           'username' => 'postgres',
-          'password' => 'some_password',
+          'password' => 'toor',
           'host' => 'localhost',
           'port' => '',
           'driver' => 'pgsql',
